server/www/fountain.js

```diff
@@ -0,0 +1,33 @@
+var of;
+const button = document.querySelector('input');
+const paragraph = document.querySelector('p');
+
+
+button.addEventListener('click', updateButton);
```

Comment:
u're trying to attach a function called `updateButton` to the click event of that button... but i don't see that u've actually created this function anywhere?